.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "semi": true,
   "singleQuote": false,
```

Comment:
We've got `singleQuote = true` on the internal NX repo, did we want to change one to match if so which one?